esrally/mechanic/launcher.py

```diff
@@ -335,8 +290,8 @@ def _start_node(self, node_configuration, node_count_on_host):
         t = telemetry.Telemetry(enabled_devices, devices=node_telemetry)
         env = self._prepare_env(car, node_name, java_home, t)
         t.on_pre_node_start(node_name)
-        node_process = self._start_process(env, node_name, binary_path)
-        node = cluster.Node(node_process, host_name, node_name, t)
+        node_pid = self._start_process(env, node_name, binary_path)
```

Comment:
The parameters do not match the signature of `_start_process`. I noticed this because Rally failed when I attempted to test this locally with:

```
esrally --test-mode --distribution-version=1.7.6 --on-error=abort --challenge=append-no-conflicts-index-only
```